src/components/CustomTable/CustomisedTableHead/index.js

```diff
@@ -0,0 +1,3 @@
+import CustomisedTableHead from "./CustomisedTableHead"
+
+export default CustomisedTableHead
```

Comment:
You can simplify this to 

```
export { default } from "./CustomisedTableHead";

```

The same can be done in your other index.js filess for your components